### PR TITLE
Add multi account support

### PR DESCRIPTION
**Pull request by isubas**
I've made a small addition to manage multiple dropbox accounts. 
This insertion is a minimum working patch.

### CODE REVIEW

**Comment by anishathalye** [comment]
What's the use case for multiple accounts?

**Comment by isubas** [comment]
I have two dropbox accounts I'm using. One of these is the one the software team uses in common, and my personal account. I use it for projects on both accounts. With multi-account support I want to meet these needs from a single configuration file. This change is also preserved in the old configuration.

File: git_remote_dropbox/__init__.py [hunk 1]
```diff
@@ -692,9 +692,10 @@ def __getitem__(self, key):
 def main():
     # configure system
     stdout_to_binary()
-
     name, url = sys.argv[1:3]
-    url = url.lower()
+    url_parse = urlparse(url.lower())
+    token_key = url_parse.username or 'token'
+    url = url_parse.scheme + '://' + url_parse.hostname + url_parse.path
```
Comment (anishathalye) [inline git_remote_dropbox/__init__.py]: This seems like a hacky way of doing things - why not just use URL parsing for the whole thing, then? Why reassemble this and then use string parsing?

File: README.rst [hunk 2]
```diff
@@ -49,7 +57,8 @@ Setup
 .. code:: json
 
     {
-        "token": "xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxx"
+        "token": "xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxx",
+        "username": "yyyyyyyy-yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy-yyyyyyyyyyyyyyyyyyyyy"
```
Comment (anishathalye) [inline README.rst]: I think it might be better to leave the examples as-is and add a separate section about multi-account support.

**Comment by scollovati** [comment]
Hi @isubas is this insertion working with multiple token generated by the same account (different APP - different folders) or only different Dropbox Accounts?

What should one write as "username" associated with the token?
Thanks!

**Comment by anishathalye** [comment]
See current `develop` branch.

**Pull request closed without merging**